### ISSUE TITLE
Simply including Fresco in build.gradle causes app to crash

### ISSUE THREAD

**Issue by daerimin** [facebook/fresco#1493]
I'm anxious to try this library, but I can't even begin to because simply adding:

compile 'com.facebook.fresco:fresco:0.14.1'

to the build.grade file prevents my app from starting.  I haven't even made any references to it anywhere yet.  Here's the stack trace:

java.lang.NoSuchMethodError: No static method setOnApplyWindowInsetsListener(Landroid/view/View;Landroid/support/v4/view/OnApplyWindowInsetsListener;)V in class Landroid/support/v4/view/ViewCompatLollipop; or its super classes (declaration of 'android.support.v4.view.ViewCompatLollipop' appears in /data/data/com.[redacted]/files/instant-run/dex/slice-internal_impl-24.2.1_6913cfa4450c0119730608c0340b7994e4332ef0-classes.dex)
at android.support.v4.view.ViewCompat$LollipopViewCompatImpl.setOnApplyWindowInsetsListener(ViewCompat.java:1619)
        at android.support.v4.view.ViewCompat.setOnApplyWindowInsetsListener(ViewCompat.java:2924)
        at android.support.v7.app.AppCompatDelegateImplV7.createSubDecor(AppCompatDelegateImplV7.java:425)
        at android.support.v7.app.AppCompatDelegateImplV7.ensureSubDecor(AppCompatDelegateImplV7.java:312)
        at android.support.v7.app.AppCompatDelegateImplV7.setContentView(AppCompatDelegateImplV7.java:277)
        at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:140)
        at com.[redacted].MainActivity.onCreate(MainActivity.java:138)
        at android.app.Activity.performCreate(Activity.java:6237)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1107)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2369)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476)
        at android.app.ActivityThread.-wrap11(ActivityThread.java)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:148)
        at android.app.ActivityThread.main(ActivityThread.java:5417)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)

It seems by the stack that the presence of the v4 support library is causing the problem.  Could that be?  If so, is this a show-stopper for Fresco?  I use the DrawerLayout and the ViewPager from that library.

Thanks for your time!

**Comment by oprisnik** [comment]
I guess you are using an older version of the support library? Try upgrading to the one we use for Fresco, [24.2.1](https://github.com/facebook/fresco/blob/master/gradle.properties#L42).

**Comment by manuelgon47** [comment]
Yes, I fix it updating support library version to 24.2.1 in my build.gradle file:
```
    compile 'com.android.support:appcompat-v7:**24.2.1**'
    compile "com.android.support:support-v13:**24.2.1**"

```

**Comment by oprisnik** [comment]
Good to hear that it works now!
